client/src/pages/TaskFormPage/TaskFormPage.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styles from "./TaskFormPage.module.css";
+import { InputBox } from "../../components/InputBox/InputBox";
+import { CredentialsButton } from "../../components/CredentialsButton/CredentialsButton.js";
+import CloseButton from "react-bootstrap/CloseButton";
+
+const TaskFormPage = () => {
+  return (
+    <div className={styles.bg}>
+      <CloseButton variant="white" />
+      <p className={styles.Header}>Create a Task</p>
+      <div className={styles.inputBox}>
+        <InputBox header="Task Title" type="text" placeholder="Task Name" />
+        <InputBox header="Location" type="text" placeholder="Location Name" />
+        <InputBox
+          header="Description"
```

Comment:
Although having the fields like this does work, it would be better to have it such as `header={"Description"}` (WIth the curly braces)

Reply:
Please make the changes with the braces to the rest of the fields on this file